src/town_gui.cpp

```diff
@@ -258,6 +260,13 @@ struct TownAuthorityWindow : Window {
 	virtual void OnClick(Point pt, int widget, int click_count)
 	{
 		switch (widget) {
+			case WID_TA_ZONE_BUTTON: {
```

Comment:
You're not declaring any variables in this block, so it doesn't technically need the braces

Reply:
I will correct it in the next version, thanks.

Reply:
Grown fond of the braces, so I declared a variable inside.